docs/installation.rst

```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
 
     $ pip install --ignore-installed --upgrade tiatoolbox
 
+If you already have setup a Python environment with all the pre-requisite software and dependencies installed and you would like to keep the existing versions of these dependencies, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install --no-deps tiatoolbox
+
 If you don't have `pip`_ installed, this `Python installation guide`_ can guide
 you through the process.
 
 .. _pip: https://pip.pypa.io
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or miniconda), you can install TIA toolbox using the following command:
```

Comment:
```suggestion
After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or miniconda / mamba), you can install TIA toolbox using the following command:
```

Reply:
Mamba: https://mamba.readthedocs.io/en/latest/user_guide/mamba.html

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/187f81540e45c1faae08649dcc710a4090812eda